cmd/kube-apiserver/app/options/options.go

```diff
@@ -44,6 +44,7 @@ type APIServer struct {
 	AuthorizationMode          string
 	AuthorizationConfig        apiserver.AuthorizationConfig
 	BasicAuthFile              string
+	DefaultStorageMediaType    string
```

Comment:
This should go in genericapiserver rather than duplicating in kubernetes-apiserver and federation-apiserver?